### ISSUE TITLE
Naming - `TableSchema::columns()`

### ISSUE THREAD

**Issue by arogachev** [yiisoft/db#666]
https://github.com/yiisoft/db/blob/857f0a855bb22e5c230114578baf585d50458578/tests/Db/Schema/SchemaTest.php#L473-L477

Singular form probably fits better.

**Comment by samdark** [comment]
Indeed, it accepts a single column https://github.com/yiisoft/db/blob/857f0a855bb22e5c230114578baf585d50458578/src/Schema/TableSchemaInterface.php#L129. Also, I'd rename $index -> $name.
